lib/Model/Listing/AbstractListing.php

```diff
@@ -571,4 +571,14 @@ public function rewind()
         $this->getData();
         reset($this->data);
     }
+
+    public function count()
```

Comment:
phpdoc

Reply:
Added return type. Anything more necessary? As this comes from the interface I would expect that people will lookup on https://www.php.net/manual/en/class.countable.php for more verbose description.